lib/cmds/update.ts

```diff
@@ -25,6 +25,7 @@ let prog = new Program()
     .addOption(Opts[Opt.ALTERNATE_CDN])
     .addOption(Opts[Opt.STANDALONE])
     .addOption(Opts[Opt.CHROME])
+    .addOption(Opts[Opt.IOS])
```

Comment:
Check to see if the user is on os.type() === 'Darwin' before adding the iOS options.